webpack.config.js

```diff
@@ -202,7 +202,7 @@ const webpackConfig = {
 			} ),
 			TranspileConfig.loader( {
 				workerCount,
-				configFile: path.resolve( __dirname, 'babel.dependencies.config.js' ),
```

Comment:
This is specifically for `node_modules/` though? :grimacing:

Reply:
Just experimenting, will probably throw this away. What are you even doing looking at it!?